setup.sh

```diff
@@ -47,7 +47,7 @@ else
    fi
 fi
 
-pip install django>=2.1 pillow grandalf django-prepared-query appdirs
+pip install "django>=2.1" pillow grandalf "django-prepared-query" appdirs "psycopg2-binary"
```

Comment:
technically, the default setup doesn't require psycopg2, this is only for those advanced users that want to use Postgres.
On the other hand, there's not much of a cost in requiring an extra dependency (in fact, django-prepared-query itself is not really needed -- I had wanted to use it to speed things up, but it didn't quite work out)

Reply:
I weighed the extra dependency against the convenience, at least during this development phase, and thought it might be worth including without significant impact. I wondered about the django-prepared-query bit.

Though you should see what I've got in my local `pip install`. I've been playing with `channels`, `django-rest-framework`, `django-debug-toolbar` and a few other toys, though nothing concrete just yet.